src/main/kotlin/com/cognifide/gradle/aem/common/instance/tail/Log.kt

```diff
@@ -58,42 +60,38 @@ class Log(
 
         private const val LOGS_SEPARATOR = "  "
 
-        private val DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy HH:mm:ss.SSS")
-
         private val PRINT_DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss.SSS")
 
-        fun create(info: LogInfo, logLines: List<String>): Log {
+        fun create(aem: AemExtension, info: LogInfo, logLines: List<String>): Log {
+
             if (logLines.isEmpty() || logLines.first().isBlank()) {
                 throw TailerException("Passed log entry is empty!")
             }
 
             val fullLog = logLines.joinToString("\n")
+            val result = matchLogLine(logLines.first())?.groups ?: throw TailerException("Nothing to parse")
```

Comment:
previously when line was not matched user was seeing 

```
throw TailerException("Passed text is not a log entry\nPattern:\n$LOG_PATTERN\nText:\n${logLines.first()}")
```

now will see mysterious `Nothing to parse` without any context which is bad